FD 1.17.0: Documentation other than the release notes (#135)
mulesoft/docs-design-center: modules/ROOT/pages/fd-connection-view-audit-logs.adoc

= View Audit Logs for Connections That are Shared
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords:

:imagesdir: ../assets/images

Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.

To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:

* The time and date of the action
* The product in which the action was taken (i.e., Flow Designer)

> * The type of change, which is always "Connection"

Reviewer: I'm not sure if the "type of change" is the best definition. I think it's more like "type of audit object", which in this case is a Connection. What do you think?

Author: I changed this to "type of object audited".

* The type of action: Create, Edit, or Delete,

> * The type of connector that the connection was for, such as the Database connector.

Reviewer: I'm not sure about the "Type of connector" phrase, I mean, you can see it, but only downloading the payload. 
In the main table, in the column "Object" you will see the "name" of the connection, 

that name is written by the user so, no necessarily you will see a name related to the type of connector...

Author: I misread the text that appears in this column in the GIF that you sent me. The text is "Amazon S3 connection," but I read it as "Amazon S3 connector."
Changed.

> * The user ID that was used to take the action

Reviewer: Maybe, I'll add that you can download the payload in the last column, that contains more detailed information about the connection... not credentials, but the ID of the connection, the user that created the connection, creation and modification date, type of connector, among others

Author: Added.

> = View Audit Logs for Connections That are Shared

Reviewer 2: ```suggestion
= View Audit Logs for Connections That Are Shared
```

ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords:

:imagesdir: ../assets/images

Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.

To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:

* The time and date of the action

> * The product in which the action was taken (i.e., Flow Designer)

Reviewer 2: ```suggestion
* The product in which the action was taken (for example, Flow Designer)
```

Author: "i.e." means "that is to say." It doesn't mean "for example."

> * The type of object audited, which is always "Connection"

Reviewer 2: ```suggestion
* The type of object audited, which is always Connection
```
GLOBALLY avoid the use of quotation marks. If init. caps. is not enough to distinguish a value aside from surrounding narrative, consider bold for GUI labels or mono for syntax/CLI values.

* The type of action: Create, Edit, or Delete
* The name of the connection
* The user ID that was used to take the action

> You can also download the payload for an entry in the audit log. A payload for the creation, modification, or deletion of a connection includes the ID of the connection, the ID of the user that created the connection, the type of connector that the connection was created for, and more.

Reviewer 2: "and more" always makes me wonder what more, how much more. You can get around this by identifying representational examples using "such as" or "like":
```suggestion
You can also download the payload for an entry in the audit log. A payload for the creation, modification, or deletion of a connection includes information such as the ID of the connection, the ID of the user that created the connection, and the type of connector that the connection was created for.
```

> == Procedure

Reviewer 2: Note inconsistency between using bold inline heads for this and this subtitle.
I think we should avoid inline headings.
I would like to propose gerunds for topic sets like these (Viewing Audit Logs) followed by a short statement and then a titled Concept subtopic and an imperative-titled subtopic (View Audit Logs).

. In Anypoint Platform, open Access Management.

> . Select *Audit Logs* in the left pane.

Reviewer 2: Do you think we can start calling this "the navigation pane" or "the navigation menu"?

> . On the *Audit Logs* page, select *Flow Designer* in the *Products* filter and *Connector* in the *Types* filter.

Reviewer 2: I don't know why, but this seemed an abrupt ending to me. I wonder if we should include a screen capture?
